Add details on including ~/.local/bin in PATH

sorieux opened the pull request:
Added instructions for creating the ~/.local/bin directory if it doesn't exist and for adding it to the user's PATH environment variable. This update aims to help new users with setting up their environment for custom scripts and executables.

maxmouchet commented:
Thanks @sorieux! Can you add this at the end of the readme, and maybe rename the section to "Help"? I'd like to keep the Installation & Usage sections on top :)

sorieux commented:
Absolutely, no problem @maxmouchet . Indeed, it does make the README more coherent and clear. I've moved the section to the end and refactored the instructions.

maxmouchet commented:
Perfect, thanks! :)
